intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/InvalidDescriptorErrors.kt

```diff
@@ -23,163 +26,156 @@ class PropertyWithDefaultValue(
-      "For detailed info refer to https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html"
+  sinceBuild: IdeVersion
+) : InvalidDescriptorProblem(
+  descriptorPath = descriptorPath,
+  detailedMessage = "The <since-build> parameter ($sinceBuild) does not match the multi-part build number format " +
+                    "<branch>.<build_number>.<version>, for example, '182.4132.789'."
+) {
+  override val hint = ProblemSolutionHint(
+    example = "since-build=\"182.4132.789\"",
+    documentationUrl = "https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html"
+  )
 
   override val level: Level
     get() = Level.ERROR
 }
 
 class ErroneousUntilBuild(
   descriptorPath: String,
-  val untilBuild: IdeVersion
-) : InvalidDescriptorProblem(descriptorPath) {
-  override val detailedMessage: String
-    get() = "until build '$untilBuild' must match the multi-part build number format, for example '182.4132.789' or '182.*'. " +
-      "For detailed info refer to https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html"
+  untilBuild: IdeVersion
+) : InvalidDescriptorProblem(
+  descriptorPath = descriptorPath,
+  detailedMessage = "The <until-build> parameter ($untilBuild) does not match the multi-part build number format " +
+                    "<branch>.<build_number>.<version>, for example, '182.4132.789'."
+) {
+  override val hint = ProblemSolutionHint(
+    example = "until-build=\"182.4132.789\"",
+    documentationUrl = "https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html"
+  )
 
   override val level: Level
     get() = Level.ERROR
 }
 
-class ProductCodePrefixInBuild(descriptorPath: String) : InvalidDescriptorProblem(descriptorPath) {
-  override val detailedMessage: String
-    get() = "'since-build' and 'until-build' shouldn't contain product code prefix"
-
+class ProductCodePrefixInBuild(descriptorPath: String) : InvalidDescriptorProblem(
+  descriptorPath = descriptorPath,
+  detailedMessage = "The <since-build> and <until-build> parameters must not contain product code prefix."
+) {
   override val level: Level
     get() = Level.ERROR
 }
 
-class XIncludeResolutionErrors(descriptorPath: String, private val details: String) : InvalidDescriptorProblem(descriptorPath) {
-  override val detailedMessage: String
-    get() = "failed to resolve <xi:include>. ${details.capitalize()}"
-
-  override val level
-    get() = Level.ERROR
-}
-
-class TooLongPropertyValue(
-  descriptorPath: String,
-  private val propertyName: String,
-  private val propertyValueLength: Int,
-  private val maxLength: Int
-) : InvalidDescriptorProblem(descriptorPath) {
-  override val detailedMessage: String
-    get() = "value of property '$propertyName' is too long. Its length is $propertyValueLength which is more than maximum $maxLength characters long"
-
-  override val level
-    get() = Level.ERROR
-}
-
-class DefaultDescription(descriptorPath: String) : InvalidDescriptorProblem(descriptorPath) {
```

Comment:
I see that it's used in `LoggingPluginVerificationReportageAggregatorTest`, but it seems there are no changes there in this review or I miss something

Reply:
To change something in verifier we need to release lib first, because of the structure of the project.

Reply:
`LoggingPluginVerificationReportageAggregatorTest` can be fixed with the other type of the plugin problem, so no worries there.